go/cmd/dolt/commands/sql.go

```diff
@@ -570,8 +546,13 @@ func execMultiStatements(
 		return errhand.VerboseErrorFromError(err)
 	}
 
-	// Add root client
-	sqlCtx.Session.SetClient(sql.Client{User: "root", Address: "%", Capabilities: 0})
+	// check if user exists, else add them as superuser; might need to do something else for sql-server
+	if user := se.GetUnderlyingEngine().Analyzer.Catalog.MySQLDb.GetUser(config.ServerUser, config.ServerHost, false); user == nil {
+		se.GetUnderlyingEngine().Analyzer.Catalog.MySQLDb.AddSuperUser(config.ServerUser, config.ServerPass)
+	}
```

Comment:
Are these functions called from `sql-server`? I'm fairly confident that these are not, so we shouldn't have to worry about that context.